plaid/api/sandbox.py

```diff
@@ -33,9 +36,17 @@ def create(self,
         :param  str     webhook:
         '''
         options = _options or {}
+        transaction_options = {}
+        transaction_options.update(options.get('transactions', {}))
```

Comment:
super nit: Can we group all lines that ultimately end up in `options['transactions']` together one place (rn webhook stuff is getting inbetween)?

Reply:
:zap: